### DISCUSSION TITLE
How do I disable 'never combine taskbar buttons" for this?

### DISCUSSION THREAD

**Discussion by EasonTang1** [valinet/ExplorerPatcher#1872] [Q&A] [answered]
Opened programs on my taskbar take up ~5 icon's worth of space. How do I disable this? I installed this earlier and the .exe didn't have a GUI or anything to interact with to customize the installation...

**Comment by pyrates999** [comment]
https://www.prajwaldesai.com/never-combine-taskbar-buttons-in-windows-11

**Reply by EasonTang1** [reply]
hello, that setting doesn't appear for me. 
![image](https://github.com/valinet/ExplorerPatcher/assets/84527360/cf3c5f4b-89d3-48ee-bd2f-5946942252df)

**Reply by EasonTang1** [reply]
my windows version was updated yesterday - version 10.0.22621 build 22621

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

**Reply by EasonTang1** [reply]
22621.2134 for windows and 22621.1992.56.1 for EP

**Comment by pyrates999** [comment, marked as answer]
Open the explorer patcher properties application and go to taskbar on the left side.  You should see entries for combine taskbar items and can disable it.

**Reply by EasonTang1** [reply]
![image](https://github.com/valinet/ExplorerPatcher/assets/84527360/c5f71363-a9a6-4baa-9c58-4ff7c8b0fbe5)
where is taskbar?

**Reply by tallman69** [reply]
Right click on an open space in the Taskbar.  Click on `Properties`.  That will open the ExploerePatcher settings.  Select `Taskbar` at the top of the lefthand column.  Towards the bottom on the righthand column you will find `Combine taskbar icons`.  Click on that and choose the behavior you want.
